docs/pages/index.md

```diff
@@ -26,14 +26,18 @@ Hardware specifications minimum / recommended, to run the Lodestar client.
 |           | Minimum                                | Recommended                            |
 | --------- | -------------------------------------- | -------------------------------------- |
 | Processor | Intel Core i3–9100 or AMD Ryzen 5 3450 | Intel Core i7–9700 or AMD Ryzen 7 4700 |
-| Memory    | 16GB RAM                               | 32GB RAM                               |
-| Storage   | 100GB available space SSD              | 1TB available space SSD                |
-| Internet  | Broadband connection                   | Broadband connection                   |
+| Memory    | 8 GB RAM                               | 16 GB RAM                              |
+| Storage   | 130 GB available space SSD             | 200 GB available space SSD             |
+| Internet  | Reliable broadband with 10mbps upload  | Reliable broadband with >10mbps upload |
 
-## About these docs
+### Execution Client
 
-This documentation is open source, contribute at [Github Lodestar repository /docs](https://github.com/ChainSafe/lodestar/tree/unstable/docs).
+If you run the [execution client](https://ethereum.org/en/developers/docs/nodes-and-clients/#execution-clients) on the same host, you will need to check their requirements and add them to the above requirements.
```

Comment:
Shall we also add a hardware expectation for running EL+CL like what Nimbus does here? 
```Broadly, to run both an execution and a consensus client on the same machine, we recommend a 2 TB SSD and 16 GB RAM.```

https://nimbus.guide/hardware.html

Reply:
I opted not to only because of the variations of the different EL clients. Will be easier IMO to just have the user figure out their recommendations and add it to our recommendations.

Reply:
> Broadly, to run both an execution and a consensus client on the same machine, we recommend a 2 TB SSD and 16 GB RAM.

I am not sure this is even a good recommendation anymore, I generally recommend people to go for 4TB nowadays since it's unknown when we will ship the verge + blob count increase adds further storage burden.

16GB is also bit tight for Lodestar with most other ELs, it's not sufficient with Nethermind for example, my server is sitting at 17.2GB